<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="angle_label">
         <property name="text">
          <string>Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="angle_input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="angleRandom_label">
         <property name="text">
          <string>Angle random</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="angleRandom_input">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="axiom_label">
         <property name="text">
          <string>Axiome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="axiom_input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="rules_label">
         <property name="text">
          <string>Règles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="rules_input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="iterations_label">
         <property name="text">
          <string>Itérations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterations_input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="length_label">
         <property name="text">
          <string>Longueur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="length_input">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="longueurRandom_label">
         <property name="text">
          <string>Longueur random</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="longueurRandom_input">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="reductionRadius_label">
         <property name="text">
          <string>Réduction radius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="reductionRadius_input">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="radiusBranche_label">
         <property name="text">
          <string>Radius branche</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="radiusBranche_input">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="radiusBrancheRandom_label">
         <property name="text">
          <string>Radius branche random</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="radiusBrancheRandom_input">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
